archive/ff-history.html

```diff
@@ -446,10 +446,10 @@ <H2>
   svg) fonts.
   <P>
   In June I was thinking of the embolden command I did the year before, and
-  realized that that was essentially the same idea as was needed for generating
+  realized that was essentially the same idea as was needed for generating
```

Comment:
Word repetition is not always incorrect grammatically. In fact, the new line here makes no sense. All changes must be read by a native speaker and verified, especially places "that" is duplicated.